features/step_definitions/session_steps.rb

```diff
@@ -146,7 +150,7 @@
 end
 
 Then "(I )(they )should not be logged in" do
-  assert_includes [provider_sessions_path, session_path], current_path
+  assert_current_path %r{\A(/p/sessions|/session)\z}, ignore_query: true
```

Comment:
`assert_current_path` will wait for the ongoing requests to complete.

Reply:
They can be not logged in and be in some other URLs

Reply:
> Fine, you can consider adding also `/p/signup` and `signup`.

I added this to https://issues.redhat.com/browse/THREESCALE-11890

It seems that it's not a must for current cucumber steps.